test/src/main/java/org/springframework/security/test/web/servlet/request/SecurityMockMvcRequestBuilders.java

```diff
@@ -124,6 +137,22 @@ public LogoutRequestBuilder logoutUrl(String logoutUrl, Object... uriVars) {
 
 		private LogoutRequestBuilder() {
 		}
+
+		@Override public boolean isMergeEnabled() {
+			return true;
+		}
+
+		@Override public Object merge( Object parent ) {
+			if (parent == null) {
+				return this;
+			}
+			if (parent instanceof MockHttpServletRequestBuilder) {
```

Comment:
I think we can just check if it is an instance of `Mergeable` since that is what we are casting it to. The check for what we pass in is a `MockHttpServletRequestBuilder` will be done when we invoke `MockHttpServletRequestBuilder.merge`